src/ripple/app/misc/TxQ.h

```diff
@@ -862,18 +862,17 @@ template <class T>
 XRPAmount
 toDrops(FeeLevel<T> const& level, XRPAmount const& baseFee)
 {
-    if (auto const drops = mulDiv(level, baseFee, TxQ::baseLevel); drops.first)
-        return drops.second;
+    if (auto const drops = mulDiv(level, baseFee, TxQ::baseLevel))
+        return *drops;
 
     return XRPAmount(STAmount::cMaxNativeN);
 }
 
 inline FeeLevel64
 toFeeLevel(XRPAmount const& drops, XRPAmount const& baseFee)
 {
-    if (auto const feeLevel = mulDiv(drops, TxQ::baseLevel, baseFee);
-        feeLevel.first)
-        return feeLevel.second;
+    if (auto const feeLevel = mulDiv(drops, TxQ::baseLevel, baseFee))
+        return *feeLevel;
 
     return FeeLevel64(std::numeric_limits<std::uint64_t>::max());
```

Comment:
You could probably use `.value_or()` here as well

Reply:
Can you use instead:

`return mulDiv(drops, TxQ::baseLevel, baseFee).value_or(ripple::muldiv_max);`